src/handlers/poap.ts

```diff
@@ -0,0 +1,14 @@
+import { rainbowFetch } from '../rainbow-fetch';
+import { parsePoaps } from '@rainbow-me/parsers';
+import logger from 'logger';
+
+export const fetchPoaps = async (address: string) => {
+  try {
+    const url = `https://api.poap.xyz/actions/scan/${address}`;
+    const data = await rainbowFetch(url, {});
+    return parsePoaps(data);
+  } catch (error) {
+    logger.log('Error getting POAPs', error);
+    throw error;
```

Comment:
if you throw here it will bubble up and break all the opensea NFTs.
You wanna captureException and swallow

Reply:
false 🐻 but will capture for sentry